eos/form-factors/analytic-b-to-psd-dkmmo2008-impl.hh

```diff
@@ -321,6 +326,8 @@ namespace eos
         std::function<double (const double &)> rescale_factor_p;
         std::function<double (const double &)> rescale_factor_0;
         std::function<double (const double &)> rescale_factor_T;
+        SpecifiedOption opt_decay_constant;
```

Comment:
I understand the usage, but can it be documented for the future?

Reply:
Yes, a comment here would be approriate.

Reply:
Fixed.